### PR TITLE
Feat(web): Switch Demo, Grid & Container to v3 design tokens structure

### PR DESCRIPTION
**Pull request by crishpeen**

### CODE REVIEW

**Comment by literat** [comment]
I think it would be better to refer in the commits to `the v3
 token structure` instead of just `new tokens`. Also, maybe using `Switch <component> to` or `Support the v3 design token structure` is better than just `Make component work again`. What do you think? :-)

**Comment by crishpeen** [comment]
> I think it would be better to refer in the commits to `the v3 token structure` instead of just `new tokens`. Also, maybe using `Switch <component> to` or `Support the v3 design token structure` is better than just `Make component work again`. What do you think? :-)

Good point. IMHO something like `switch` is good, `support` feels like we still support the old system, but we don't.

I will change it.

File: tests/e2e/demo-components-compare.spec.ts
Comment (adamkudrna) [inline tests/e2e/demo-components-compare.spec.ts:1]: 👍🏻

File: apps/demo/partials/header.hbs [hunk 1]
```diff
@@ -1,8 +1,8 @@
-<header class='bg-inverted text-secondary-inverted py-1000 py-tablet-1100'>
+<header class='spirit-theme-light-inverted bg-primary text-secondary py-1200 py-tablet-1400'>
   <div class='Container'>
 
-    <h1 class='typography-heading-xlarge-text'>Spirit Design System</h1>
```
Comment (adamkudrna) [inline apps/demo/partials/header.hbs:4]: Hmm, do we have a migration guide for the text styles? 🤔

File: apps/demo/partials/layout/default.hbs [hunk 2]
```diff
@@ -9,7 +9,7 @@
     {{> header package=""}}
   {{/with}}
 
-  <main class="py-900 py-tablet-1200">
```
Comment (adamkudrna) [inline apps/demo/partials/layout/default.hbs:12]: Ooh, I didn't know there are so many new spacings 😮.

File: packages/demo/scss/index.scss [hunk 3]
```diff
@@ -23,7 +23,7 @@
     gap: 0.5rem 0.75rem;
     align-items: baseline;
     margin-bottom: 1.5rem;
-    font-size: 1.125rem; /* Based on the `heading-small-text` style. */
+    font-size: 1.25rem; /* Based on the `heading-4-bold` style. */
```
Comment (adamkudrna) [inline packages/demo/scss/index.scss:26]: Shouldn't it be a different heading style then so the font size remains unchanged?

**Review by adamkudrna — APPROVED** [review]

**Comment by adamkudrna** [comment]
Please add the label to run visual tests here when the PR is ready.

**Review by literat — APPROVED** [review]

File: apps/demo/partials/header.hbs (hunk 1 above)
Comment (crishpeen) [reply apps/demo/partials/header.hbs:4]: Not yet, feel free to create one 🙃

File: packages/demo/scss/index.scss (hunk 3 above)
Comment (crishpeen) [reply packages/demo/scss/index.scss:26]: There is no style with the same values. But this is only on mobile 🤷

**Review by pavelklibani — APPROVED** [review]

**Pull request merged**